### PR TITLE
Translate 'reviewprocess.md' into ko

### PR DESCRIPTION
**Pull request by mukmookk**
세부 내용은 다음과 같습니다.
1. policies의 reviewprocess.md에 대한 한국어 번역을 수행
2. 해당 문서에 대한 사이드바 변경

Closes: #137

### CODE REVIEW

File: docs/policies/reviewprocess.md [hunk 1]
```diff
@@ -1,171 +1,168 @@
...
 permalink: policies_reviewprocess.html
 folder: policies
 sidebar: general_sidebar
 ---
 
-## Introduction
+## 소개
 
-The Azure Developer Platform (ADP) Architecture Review Board is a board of language architects specializing in Java, Python, TS/JS, C#, C, C++, Go, Android, and iOS.
+Azure Developer Platform (ADP) 아키텍처 리뷰 보드는 Java, Python, TS/JS, C#, C, C++, Go, Android, iOS에 전문화된 언어 아키텍트들의 보드입니다.
```
Comment (ianychoi) [inline docs/policies/reviewprocess.md:10]: For this sentence, I think "위원회" is better than "보드" for the translation of Board.

File: docs/policies/reviewprocess.md [hunk 2]
```diff
@@ -1,171 +1,168 @@
...
 
-**The Architecture Board reviews Track 2 libraries only**. By definition, a Track 2 library is one that follows our [Track 2 library design guidelines and specific language guidelines](https://azure.github.io/azure-sdk/general_introduction.html). This means that libraries produced solely by a code generator do NOT follow these guidelines; engineers MUST build a layer on top of the generated code in order to produce a library that meets the guidelines. We have developed the SDK Development Training program to help you be successful in building these libraries - find out more [here!](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/346/Azure-SDK-Development-Training)
+아키텍처 보드는 오직 Track 2 라이브러리만을 검토합니다. 정의상, Track 2 라이브러리는 우리의 Track 2 라이브러리 디자인 가이드라인과 특정 언어 가이드라인을 준수하는 라이브러리를 의미합니다. 이는 코드 생성기만으로 제작된 라이브러리는 이러한 가이드라인을 따르지 **않음**을 의미하며, 엔지니어들은 이 가이드라인을 만족시키는 라이브러리를 제작하기 위해 생성된 코드 위에 레이어를 **반드시** 구축해야 합니다. 이러한 라이브러리를 만드는데 성공할 수 있도록 SDK 개발 교육 프로그램을 개발했습니다 - [여기서](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/346/Azure-SDK-Development-Training) 자세한 내용을 알아보세요!
```
Comment (ianychoi) [inline docs/policies/reviewprocess.md:12]: Same as for "보드"

Comment (ianychoi) [inline docs/policies/reviewprocess.md:12]: Typically for Korean `[여기](link)에서` would be better

File: docs/policies/reviewprocess.md [hunk 3]
```diff
@@ -1,171 +1,168 @@
...
+우리는 모든 Azure 클라이언트 라이브러리가 Microsoft에서 생산하는 다른 API (예: .NET API)에 대해 실시하는 것과 유사한 엄격한 API 리뷰를 통과하길 기대합니다. 새로운 라이브러리의 상세한 리뷰 외에도, API에 대한 **모든 변경사항**은 릴리즈 전에 특정 언어의 아키텍트가 승인해야 합니다.
 
-Note that the library review process described here is currently an **internal** one. This document is intended to clarify the process for Azure service teams looking to have their libraries reviewed.
+여기서 설명하는 라이브러리 리뷰 프로세스는 현재 **내부적인** 프로세스입니다. 이 문서는 라이브러리를 검토하려는 Azure 서비스 팀들에게 프로세스를 명확하게 하는 데 의도되어 있습니다.
 
-## API Review Process Roadmap
+## API 리뷰 프로세스 로드맵
 
-Typically, there will be a minimum of three meetings with the Architecture Board:
+아키텍처 보드와의 회의는 일반적으로 최소 세 번 이상 진행됩니다:
```
Comment (ianychoi) [inline docs/policies/reviewprocess.md:20]: Same as this for "보드"

File: docs/policies/reviewprocess.md [hunk 4]
```diff
@@ -1,171 +1,168 @@
...
+### 1. 소개 세션
 
-[Submit an issue](https://github.com/Azure/azure-sdk/issues/new/choose) to the Architecture Board to request for a meeting. If the service is pre-release and not yet publicly disclosed, use the private repository ([azure-sdk-pr](https://github.com/Azure/azure-sdk-pr)). After creating the issue, email the [Architecture Board](mailto:adparch@microsoft.com) to communicate specific requests such as scheduling, invite lists, etc.
+이 세션은 순수하게 정보 제공/교육을 위한 것으로, 보드가 서비스와 라이브러리/새로운 기능에 대해 속도를 내는 데 도움이 됩니다. 이를 통해 초기 피드백을 받을 수 있으며, 서비스 디자인에 영향을 줄 수 있습니다. API 네임스페이스, 함수 이름, 타입 등의 고수준 주제가 이 첫 번째 논의에서 제안될 것입니다.
 
-Depending on the type of review, use the appropriate template to create the issue. Ensure that you included everything the issue template asks for and in the correct format.
+**준비물 (리뷰 요청을 위한 GitHub 이슈에 포함시켜야 하는 내용):**:
+* 서비스 소개 자료
+    * 서비스를 소개/설명하는 문서 링크
+    * 일부 팀들은 PowerPoint 소개를 준비하기도 합니다
```
Comment (ianychoi) [inline docs/policies/reviewprocess.md:55]: Period(.) seems missing

**Review by ianychoi — CHANGES_REQUESTED** [review]
Would you review my comment and reflect to revise your overall content? Thanks

File: docs/policies/reviewprocess.md (hunk 1 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md:10]: Completed.

File: docs/policies/reviewprocess.md (hunk 2 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md:12]: Completed.

Comment (mukmookk) [reply docs/policies/reviewprocess.md:12]: Completed.

File: docs/policies/reviewprocess.md (hunk 3 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md:20]: Completed.

File: docs/policies/reviewprocess.md (hunk 4 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md:55]: Completed.

File: docs/policies/reviewprocess.md (hunk 1 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md]: Completed.

File: docs/policies/reviewprocess.md (hunk 2 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md]: Completed.

Comment (mukmookk) [reply docs/policies/reviewprocess.md]: Completed.

File: docs/policies/reviewprocess.md (hunk 4 above)
Comment (mukmookk) [reply docs/policies/reviewprocess.md]: Completed.

**Pull request merged**